lib/allthenews_app.dart

```diff
@@ -1,18 +1,13 @@
 import 'package:allthenews/ui/common/theme/theme.dart';
 import 'package:allthenews/ui/pages/news_list/news_list_page.dart';
-import 'package:allthenews/ui/pages/news_list/secondary_news/secondary_news_list_page.dart';
 import 'package:flutter/material.dart';
 
 class AllTheNewsApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
       theme: newsTheme,
-      initialRoute: NewsListPage.id,
-      routes: {
-        NewsListPage.id: (context) => NewsListPage(),
-        SecondaryNewsListPage.id: (context) => SecondaryNewsListPage(),
-      },
+      home: NewsListPage(),
```

Comment:
jakiś konkretny powód rezygnacji z routów?

Reply:
stwierdziłem, że lepszym wyjściem jest przekazywanie argumentów poprzez konstruktor, aniżeli wyciąganie z arguments, a poprzednie rozwiązanie na to nie pozwalało - ponadto, dzięki temu nie trzeba całej nawigacji trzymać w app bo tak naprawdę to nie widzę ani jednej zalety trzymania tej mapy w ten sposób